src/pages/GetStarted/index.tsx

```diff
@@ -103,8 +103,11 @@ export class GetStarted extends React.PureComponent<Props, State> {
       this.showAlert({
         key: 'new-workspace-failed',
         variant: AlertVariant.danger,
-        title: errorMessage + '.'
+        title: e.message ? e.message : errorMessage + '.'
       });
+      if (e.message) {
+        throw new Error(errorMessage + ', \n' + e.message);
+      }
       throw new Error(errorMessage + ', \n' + e);
```

Comment:
What is `e`? If it's shown as Object[object] then it's better to show at least response code if it's available.

Reply:
I've made changes so that `e.message` will show the message from Che Server if the request succeeds or something like "Request failed with status code 409" when the request fails to Che Server fails